src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -74,6 +74,7 @@ function useChatUtilsHook(): UseChatUtilsHook {
       }
     }
 
+    refresh('channelList');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `refresh('channelList')` is a good move if the channel list needs to be updated after certain operations. However, keep in mind that this could potentially lead to performance issues if the 'channelList' is large and this operation is called frequently. Consider using more granular updates or caching strategies if performance becomes an issue.
```diff
+    refresh('channelList');
     refresh('channelInfo');
     refresh('chat');
     refresh('user');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->